src/librustdoc/html/static/main.js

```diff
@@ -1227,19 +1227,30 @@
             return [displayPath, href];
         }
 
+        function escape(content) {
+            var h1 = document.createElement('h1');
+            h1.textContent = content;
+            return h1.innerHTML;
+        }
+
+        function pathSplitter(path) {
+            return '<span>' + path.replace(/::/g, '::</span><span>');
```

Comment:
Is there a problem if this function opens a `<span>` tag without closing it?

Reply:
Good catch. It's an old bug in addition of that!